scripts/v6/ch10-07-bad-script.rsc

```diff
@@ -80,7 +80,7 @@
     } on-error={
         $LogMessageFunc ("GetWebPageFunc: unable to retrieve site: \
             $SiteName !");
-        :return { "duration"="*** test failed ***" }
+        :return { "*** test failed ***" }
```

Comment:
The original line 83 is correct, as an array with a named field needs to be returned for the array lookup in the later line: `:put ("Page d/load duration: " . $UrlFetch->"duration"); `